lib/transactional.pm

```diff
@@ -117,7 +117,7 @@ sub process_reboot {
             assert_screen 'grub2', 150;
             wait_screen_change { send_key 'ret' };
         }
-        assert_screen 'linux-login', 200;
+        assert_screen 'linux-login', 500;
```

Comment:
In general, I would prefer to set `TIMEOUT_SCALE=X` which solves the timeout for the whole test suite or even flavor if set in medium (which makes sense as this would be a problem for each test).

Reply:
Good point,  due to current situation, we have to run ppc64le image osd test of SL Mircro6.1 via QEMU with KVM, mean that there is low performance during test run.  At current DEBUG step, try to use TIME OUT way to finish all test modules for ppc64le images test run.

Reply:
Meanwhile, Try with TIMEOUT_SCALE=12 VNC_TYPING_LIMIT=3. still have TIME OUT problem.

Reply:
Try to set up TIMEOUT_SCALE=30 to solve the timeout for the whole test suite